File: playground/BOJ/boj_2675/boj_2675.js
/*
----------------------------------------------
Sub : [BOJ] 문자열 반복
Link: https://www.acmicpc.net/problem/2675
Tag : JS, 
Memo
----------------------------------------------
*/

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

// 첫 번째 줄은 테스트 케이스 개수
const T = parseInt(input[0]);

// 각 테스트 케이스 처리
for (let i = 1; i <= T; i++) {
    // 각 줄에서 R과 S를 분리
    let [R, S] = input[i].split(' ');
    R = parseInt(R);
    
    // 문자열 P 생성
    let P = '';
    for (let char of S) {
        P += char.repeat(R);
    }
    
    // 결과 출력
    console.log(P);
}

File: Online_Judge_Problems/BOJs/boj_2675/boj_2675.py
'''
----------------------------------------------
Sub : [BOJ] 문자열 반복
Link: https://www.acmicpc.net/problem/2675
Tag : Python, 
Memo
----------------------------------------------
'''

# 테스트 케이스 개수 입력
T = int(input())

# 각 테스트 케이스 처리
for _ in range(T):
    S = input()     # 문자열 입력
                    # 각 줄에서 R과 S를 분리
    R = int(S[0])   # 문자를 반복할 수 R 
    S = S[2:]       # S에서 문자열만 추출

    # 결과 출력
    for i in S:
        print(i*R, end="")
    print()

